server/views/partials/pagination.njk

```diff
@@ -0,0 +1,2 @@
+{%- from "moj/components/pagination/macro.njk" import mojPagination -%}
```

Comment:
is this nessecary ? can't the templates that want to use this just import it?

Reply:
I liked the idea of a partial page that handled the pagination as it's compartmentalised into a single place; however I didn't expect it to be a single line...

I still like the idea of leaving it as is because if there is a change that we need to do to all pagination elements then it would be easy to do by modifying this partial page...